src/api/index.ts

```diff
@@ -54,27 +50,27 @@ export interface Users {
   data: User[];
 }
 
+export type RepoSortOption =
+  | 'stars'
+  | 'forks'
+  | 'help-wanted-issues'
+  | 'updated';
```

Comment:
1. Could you elaborate more about those 2 options (help-wanted-issues` and `updated`)? From my current understanding:
- For `help-wanted-issues`, I think people just search for the repo and do the filter inside.
- For `updated`, I am not sure what's the value of this.

2. Besides, this PR is doing something out of the scope. Refactoring is welcome. But I think the better way of doing it is just to change to the following first. You can add more options later.
```
export type RepoSortOption =
  | 'stars'
  | 'forks';
```

Reply:
> Could you elaborate more about those 2 options (help-wanted-issuesandupdated`)? From my current understanding:
The type defined is from octokit.
I replace self-defined type with reference to the lib octokit.

![image](https://github.com/AttackOnMorty/github-ranking/assets/113034339/172f4518-b97d-46c8-9a1e-ca49c5ab82c1)
**Figure: https://docs.github.com/en/rest/search/search?apiVersion=2022-11-28#search-repositories**

Since we are not intended to use those two options, I excluded them from type `RepoSortOption`.